library/dnet_common.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2008+ Evgeniy Polyakov <zbr@ioremap.net>
+ * Copyright 2015+ Yandex
```

Comment:
FSF assigns copyright to itself to prevent legal risks - it requires authors to yield appropriate rights at commit time.
I follow linux kernel rule of thumb - original author holds copyright notice and it can be extended when substantial part of the file is changed, in practice this only happens when more than half of the file is rewritten from scratch and it is not a plain refactor or improvement, but indeed a new subsystem.

Recently (like a couple of years) copyright tendency started to decline, authorship can be detected and shown via version control system, so Torvalds recommends to stop writing copyright notices at all.